OverlayPlugin.Core/NetworkProcessors/LineActorControlExtra.cs

```diff
@@ -18,10 +19,10 @@ public class LineActorControlExtra
         private ILogger logger;
         private readonly FFXIVRepository ffxiv;
 
-        // Any category defined in this enum will be allowed as an emitted line
-        public enum Server_ActorControlCategory : ushort
-        {
-            SetAnimationState = 62,
+        // Any category defined in this array will be allowed as an emitted line
+        public static readonly Server_ActorControlCategory[] AllowedActorControlCategories = {
+            Server_ActorControlCategory.SetAnimationState,
+            Server_ActorControlCategory.DisplayPublicContentTextMessage
```

Comment:
I like the idea of defining these categories in a single place like `LineActorControlCommon` - but would it make sense to also define the list of allowed categories for each type of `ActorControl` line in there too?  Then adding future categories would only require changing the one file.

Reply:
I think I'd prefer to keep the allowed categories on their associated class from a code organization perspective.

Additionally, we'd have to have a static class def to attach them to in the `Common` file, we can't define a variable at the namespace level in C#.